pages/linux/trash.md

```diff
@@ -0,0 +1,23 @@
+# trash
+
+> A CLI trashcan/recycling bin.
+
+- Delete a file (send to trash):
+
+`trash-put {{path/to/file}}`
+
+- List files in trash:
+
+`trash-list`
+
+- Restore file from trash:
+
+`trash-restore`
+
+- Empty trash:
+
+`trash-empty`
+
+- Empty trash older than {{X}} days:
```

Comment:
Maybe this can be rephrased. I think it is better to be clear that files on the trash that are older than X days will be deleted. With this description it feels like the trash bins will be deleted.

Reply:
Changed, it. Take a look.

Reply:
Reads better now in my opinion. Thank you for your time 👍